rfcs/react-components/convergence/custom-styling.md

```diff
@@ -0,0 +1,390 @@
+# RFC: Custom Styling
+
+## Contributors
+
+Geoff Cox
+Mason Tejera - MADS use cases
+Ben Howell - easier recomposition
+Micah Godbolt - component CSS vars
+
+## Summary
+
+Fluent UI React v9 is built today to adopt the Fluent 2 design as its default style.
+
+Partners migrating from v0/v8 to v9 have brought up that v9's custom styling does not
+provide the same granularity of customization as previous versions.
+Partners are concerned their customers won't be able to deviate from the Fluent 2 design.
+
+Please see the appendix for partner use cases and scenarios.
+
+This RFC considers different options for adding new custom styling mechanisms or improving existing ones.
+
+## Current Theme and Style Customizations
+
+There are several mechanisms for customizing the theme and style of components in v9.
+Different mechanisms allow different scopes of customization: all components, components of a specific type, or
+individual instances of a component.
+
+### Don't
```

Comment:
(me) a random person looking into this RFC, shouldn't this be labelled as `anti-goals` or similar?, `don't` title is confusing, it seems like the intention is that `V9` should not do it, which is quite the opposite of the RFC isn't it?.